server/manager/src/main/java/org/apache/accumulo/manager/compaction/queue/CompactionJobPriorityQueue.java

```diff
@@ -201,8 +202,12 @@ public synchronized int add(TabletMetadata tabletMetadata, Collection<Compaction
     return jobsAdded;
   }
 
-  public long getMaxSize() {
-    return maxSize;
+  public int getMaxSize() {
+    return maxSize.get();
+  }
+
+  public void setMaxSize(int maxSize) {
```

Comment:
Could make these synchronized for memory consistency.  Also everythnig else is syncronized so making the set method synchronized will ensure the max size does not change for another thread in the middle of a method which makes things easier to reason about.

```suggestion
  public synchronized int getMaxSize() {
    return maxSize.get();
  }

  public synchronized void setMaxSize(int maxSize) {
```

Reply:
Added synchronized modifier in 18d2243